Makefile

```diff
@@ -51,6 +51,8 @@ minikube-delete:
 ########################################################################
 # Run
 
+all: cf-operator-apply cf-operator-wait kubecf-apply kubecf-wait cf-login
```

Comment:
Nothing guarantees that these will run in the correct order.  Please add [order-only prerequisites](https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html#Prerequisite-Types) as appropriate.

Reply:
Fixed in cac7e603fdcc330177fa48a36304a9dafc942970 by making it not make-based.